database/sql/src/main/scala/cromwell/database/slick/CallCachingSlickDatabase.scala

```diff
@@ -46,4 +46,12 @@ trait CallCachingSlickDatabase extends CallCachingSqlDatabase {
 
     runTransaction(action)
   }
+
+  override def invalidateCall(callCachingEntryId: Int)
+                               (implicit ec: ExecutionContext): Future[Unit] = {
+    import cats.syntax.functor._
+    import cats.instances.future._
+    val action = dataAccess.allowResultReuseForCallCachingEntry(callCachingEntryId).update(false)
+    runTransaction(action) void
```

Comment:
It's turning the return value into a `Unit` since the return type of the method is `Future[Unit]` ( @mcovarr would know better )